### PR TITLE
Resolve test failure due to deleted node

### PR DESCRIPTION
**Pull request by rmarianski**
@nvkelso could you review please?

### CODE REVIEW

File: test/663-combo-outdoor-landuse-pois.py [hunk 1]
```diff
@@ -34,14 +34,12 @@
 assert_has_feature(
     16, 19314, 24677, 'pois',
     { 'kind': 'beach_resort', 'min_zoom': 16 })
-  
-#https://www.openstreetmap.org/node/1500943741
+
+# Way: Terrasol Beach Resort (381817391) https://www.openstreetmap.org/way/381817391
```
Comment (nvkelso) [inline test/663-combo-outdoor-landuse-pois.py]: These should be on two different lines:

```
# https://www.openstreetmap.org/way/381817391
# Terrasol Beach Resort
```

**Comment by nvkelso** [comment]
Other than the above comment in https://github.com/mapzen/vector-datasource/pull/793/files#r62083093, LGTM.

File: test/663-combo-outdoor-landuse-pois.py (hunk 1 above)
Comment (rmarianski) [inline test/663-combo-outdoor-landuse-pois.py]: I've been putting them all on the same line for other tests :( I'll start using this pattern going forward.

**Comment by rmarianski** [comment]
Updated in 5e64b10.

**Pull request merged**